### ISSUE TITLE
"Send request" and "Send email" in workflow look scary

### ISSUE THREAD

**Issue by mal-tee** [datenanfragen/website#1024]
I got user feedback, that both buttons are bit too scary for what they do. The wording implies that pressing the button actually sends the request/mail. Maybe we should rephrase them.

1 . ![image](https://user-images.githubusercontent.com/45886910/217517557-4672fffd-42c2-443a-9d2d-0130f7e6152a.png)
 - My suggestion: "Continue to request"?
 
2. 
![image](https://user-images.githubusercontent.com/45886910/217517736-7b498961-1360-4eb3-809f-837a254ef8c5.png)
![image](https://user-images.githubusercontent.com/45886910/217517759-c0fa6c59-3af7-472c-9232-d57260c28a55.png)

- My suggestion: "Open in email client"?

**Comment by baltpeter** [comment]
> I got user feedback, that both buttons are bit too scary for what they do. The wording implies that pressing the button actually sends the request/mail.

Yeah, I can see that. In fact, we also recently got a comment in https://www.datenanfragen.de/blog/deine-dsgvo-rechte/ that also sounds like the user misunderstood the wording.

But I can't really think of a better wording right now. I especially don't like "Open in email client"—I'd argue that the concept of an "email client" isn't really something that people are familiar with (anymore?).
